src/client/terminals/codeExecution/terminalCodeExecution.ts

```diff
@@ -42,9 +45,25 @@ export class TerminalCodeExecutionProvider implements ICodeExecutionService {
         if (!code || code.trim().length === 0) {
             return;
         }
-
         await this.initializeRepl(resource);
-        await this.getTerminalService(resource).sendText(code);
+        if (code == 'deprecated') {
+            // If user is trying to smart send deprecated code show warning
+            const selection = await showWarningMessage(
+                l10n.t(
+                    `Python is unable to parse the code provided. Please
+                    turn off Smart Send if you wish to run code line by line or explicitly select code
+                    to force run. [logs](command:${Commands.ViewOutput}) for more details.`,
+                ),
+                'Switch to line-by-line',
```

Comment:
I think it would make more sense if we formatted the button this way to be explicit about what we are doing on behalf of the user. 
```suggestion
                'Disable Smart Send',
```

Reply:
Make sense, thank you :)